vm/src/obj/objobject.rs

```diff
@@ -44,6 +44,62 @@ fn object_ne(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
     objbool::not(vm, &eq)
 }
 
+fn object_lt(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
+    arg_check!(
+        vm,
+        args,
+        required = [(i, Some(vm.ctx.object())), (i2, None)]
+    );
+
+    Err(vm.new_type_error(format!(
+        "'<' not supported between instances of {} and {}",
+        i.borrow(),
+        i2.borrow()
+    )))
+}
+
+fn object_le(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
+    arg_check!(
+        vm,
+        args,
+        required = [(i, Some(vm.ctx.object())), (i2, None)]
+    );
+
+    Err(vm.new_type_error(format!(
+        "'<=' not supported between instances of {} and {}",
+        i.borrow(),
+        i2.borrow()
+    )))
+}
+
+fn object_gt(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
+    arg_check!(
+        vm,
+        args,
+        required = [(i, Some(vm.ctx.object())), (i2, None)]
+    );
+
+    Err(vm.new_type_error(format!(
+        "'>' not supported between instances of {} and {}",
+        i.borrow(),
```

Comment:
Instead of using the `Display` trait, go for `vm.py_str` to properly call the `__str__` or `__repr__` of the object.